ScreenManager.lua

```diff
@@ -522,6 +527,31 @@ function ScreenManager.joystickremoved( joystick )
     ScreenManager.peek():joystickremoved( joystick );
 end
 
+---
+-- Register to multiple LÖVE callbacks, defaults to all.
+-- @param callbacks (table) Table with the names of the callbacks to register to.
+--
+function ScreenManager.registerCallbacks( callbacks )
+    local registry = {};
+
+    if type( callbacks ) ~= "table" then
```

Comment:
I'm only checking if callbacks is a table (cause then we are gonna iterate through it and if it's not a table there is gonna be an error), I could also check if it's empty but well, I'm not even checking that it holds strings, or if they are valid love callbacks or functions so... yeah this should do.